### ISSUE TITLE
import pdf or dxf to use for alignment

### ISSUE THREAD

**Issue by davidperrenoud** [fritzing/fritzing-app#1936]
_From [irasc...@gmail.com](https://code.google.com/u/104729248032245122687/) on April 07, 2012 00:36:32_

What steps will reproduce the problem? 1. 2. 3. Attach your sketch file and/or custom part files to the bug report. If the sketch uses custom parts, save the sketch as shareable (under the file menu). What is the expected output? What do you see instead? What version of Fritzing are you using? On what operating system? Please use labels and text to provide additional information.

_Original issue: http://code.google.com/p/fritzing/issues/detail?id=1936_

**Comment by failiz** [comment]
Remove imported label add pcbview and importexport label

**Comment by KjellMorgenstern** [comment]
To clarify: This feature should help someone with an existing pdf or dxf, to use it as a blueprint to create a Fritzing project from it, right?

**Comment by failiz** [comment]
Yes. I have used this technique when there is not to much space in a device for the pcb. 
You export a DXF file from a mechanical CAD program where you have designed your device and import it to the pcb design program. Usually you have the dimensions of the pcb, mounting holes and restrictions (do not place components in this area, etc)
